package/lib/testHelpers/testSchema.js

```diff
@@ -37,6 +37,16 @@ const testSchema = new SimpleSchema({
     type: String,
     allowedValues: ['tuna', 'fish', 'salad'],
   },
+  allowedStringsSetOrArray: { // make sure test always runnable even without Set
+    type: Array,
+    optional: true,
+  },
+  'allowedStringsSetOrArray.$': {
+    type: String,
+    allowedValues: typeof Set === 'function' ?
+      new Set(['tuna', 'fish', 'salad']) :
+      ['tuna', 'fish', 'salad'],
+  },
```

Comment:
I'd rather see separate tests for Set vs Array with a polyfill loaded as necessary